ra2ce/analyses/analysis_config_wrapper/analysis_config_wrapper_base.py

```diff
@@ -55,10 +55,10 @@ def initialize_output_dirs(self) -> None:
 
         def _create_output_folders(analysis_type: str) -> None:
             # Create the output folders
-            if analysis_type not in self.config_data.keys():
+            if not hasattr(self.config_data, analysis_type):
                 return
-            for a in self.config_data[analysis_type]:
-                output_path = self.config_data["output"] / a["analysis"]
+            for a in getattr(self.config_data, analysis_type):
+                output_path = self.config_data.output_path / a.analysis
                 output_path.mkdir(parents=True, exist_ok=True)
```

Comment:
why using the `get` / `has` `attr` functions? We are now in control of our own object, therefore you could simply check for existence:
```python
if not self.config_data:
   return
for _analysis_type in self.config_data.analysis_type:
  # We know analysis_type(s) is a list, we don't care whether it's empty or not.
   ...
```

Reply:
Agree on the first 2 lines.
I use getattr beacause analysis_type is not an attribute. It has either the value "direct" or "indirect" which are the names of the attributes. That's what I need getattr for.

Reply:
Will loop over analysis